onadata/apps/logger/tests/models/test_note.py

```diff
@@ -0,0 +1,22 @@
+from onadata.apps.main.tests.test_base import TestBase
+from onadata.apps.logger.models import Instance, Note
+
+
+class TestNote(TestBase):
+
+    def setUp(self):
+        super(self.__class__, self).setUp()
+
+    def test_no_created_by(self):
+        self._publish_transportation_form_and_submit_instance()
+        instance = Instance.objects.first()
+        note = Note(
+            instance=instance,
+            instance_field="",
+            created_by=None,
+        )
+        note.save()
+        try:
+            note.get_data()
+        except AttributeError:
```

Comment:
Could we have checked that we get an empty string for `owner` instead? I don't think `try ...  except` in tests is a good pattern.